content/docs/contributing/release-process.md

```diff
@@ -32,25 +29,12 @@ following conditions:
     PR](https://github.com/jetstack/platform-board/pulls/new) with a link to
     here.
 
-5. You need to be added as an "Editor" to the GCP project
+4. You need to be added as an "Editor" to the GCP project
    [cert-manager-release](https://console.cloud.google.com/?project=cert-manager-release).
-   To check if you do have access, try opening [the Cloud Build
-   page](https://console.cloud.google.com/cloud-build?project=cert-manager-release).
-   To get the "Editor" permission on the GCP project, open a PR with your name
-   added to the maintainers list in
-   [`cert_manager_release.tf`](https://github.com/jetstack/terraform-jetstack/blob/master/cert_manager_release.tf)
-
-    ```diff
-    --- a/cert_manager_release.tf
-    +++ b/cert_manager_release.tf
-    @@ -17,6 +17,7 @@ locals {
-         var.personal_email["..."],
-         var.personal_email["..."],
-         var.personal_email["..."],
-    +    var.personal_email["mael-valais"],
-       ])
-     }
-    ```
```

Comment:
self review: Rather than update this diff (which doesn't match how this would be changed today) I removed it. It seems pretty clear to me that anyone who's able to reach the status of maintainer would be able to work out how to add themselves to the list without us having to describe the process here